go/prometheusexporter/collector/collector.go

```diff
@@ -741,14 +741,20 @@ func (c *Collector) recordLogMetrics(m *metric) {
 	switch metric {
 	case "http":
 		p["type"] = c.requestType(action)
+		q := removeAction(p)
```

Comment:
I think the name of the function is not descriptive enough

Reply:
What name would you propose? `mapWithoutActionLabel`?

Reply:
sounds good

Reply:
+1, or `copyWithoutActionLabel`

Reply:
That's better. I'll take that one.